docs/machine-learning/how-to-guides/index.md

```diff
@@ -1,9 +1,10 @@
 ---
-title: ML.NET How-to guides
+title: .NET Machine learning How-to guides - ML.NET
```

Comment:
Given that it's ML.NET, I think Learning should be capitalized. How-to can be lowercase.